xbmc/network/AirPlayServer.cpp

```diff
@@ -314,11 +314,11 @@ void CAirPlayServer::AnnounceToClients(int state)
   }
 }
 
-CAirPlayServer::CAirPlayServer(int port, bool nonlocal) : CThread("AirPlayServer")
+CAirPlayServer::CAirPlayServer(int port, bool nonlocal)
+  : CThread("AirPlayServer"), m_ServerSockets(std::vector<SOCKET>())
 {
   m_port = port;
   m_nonlocal = nonlocal;
-  m_ServerSockets = std::vector<SOCKET>();
```

Comment:
Please get rid of this initialization entirely, `m_ServerSockets` default constructor is already sufficient.